storage/azure/src/test/integration/backend/AzureServerStorage.test.ts

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This folder and class structure is a bit hard to understand -
![image](https://github.com/iTwin/object-storage/assets/70565417/af9f353f-61cf-4547-a2d7-0329d440da9a)

-  tests/backend-storage/src
  - utils (currently this folder is shared/utils, but I would suggest to move it up since it contains stuff used by storage specific and common tests)
  - common-tests (all the common tests and related classes)
    - ClientStorage.test.ts
    - ...
    - StorageIntegrationTests.ts
  - storage-specific-test-utils (your newly added tools)
    -  InternalStorageTests.ts
    - ...

Currently we have a lot of small folders with not really descriptive names. I think it is better to have less structure and refine it as codebase expands.

Reply:
I agree, changed according to your suggestions